classes/class.ilObjMumieTaskGUI.php

```diff
@@ -504,7 +526,9 @@ public function editAvailabilitySettings()
         $ilTabs->activateSubTab('availability_settings');
         $this->lng->loadLanguageModule('rep');
 
-        $this->initAvailabilitySettingsForm();
+        $disable_online_selection = !$this->object->isGradepoolSet();
+
+        $this->initAvailabilitySettingsForm($disable_online_selection);
```

Comment:
I thin we don't need to pass `disable_online_selection` as a parameter. We can just call `$this->object->isGradepoolSet()` from the function itself